modules/@apostrophecms/asset/index.js

```diff
@@ -373,17 +404,23 @@ module.exports = {
             } else {
               // The right choice with Docker if uploadfs is just the local filesystem
               // mapped to a volume (a Docker build step can't access that)
-              copyIn = fs.copyFile;
+              copyIn = fsCopyIn;
```

Comment:
Minor upgrade here to be able to cope with paths that need their parent folders created, i.e. `images/bg.png` requires that `images` be created. On the uploadfs side that is already automatic, where needed.